youtube_dl/extractor/vidello.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import (
+    clean_html
+)
+
+
+class VidelloIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?embed\.vidello\.com/[0-9]/(?P<id>[a-zA-Z0-9]+)/player\.html'
```

Comment:
* the first element of the path can have more than 1 digit
* the tail `/player\.html` isn't needed (unless there are other tails that should find different media with the same id):
```suggestion
    _VALID_URL = r'https?://(?:www\.)?embed\.vidello\.com/[0-9]+/(?P<id>[a-zA-Z0-9]+)'
```